backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```diff
@@ -71,32 +71,33 @@ public TestOrderService(
   @Transactional(readOnly = true)
   @AuthorizationConfiguration.RequirePermissionReadResultList
   public List<TestEvent> getTestResults(Person patient) {
-      return _terepo.findAllByPatient(patient);
+    assertPatientInCurrentOrg(patient.getLookupId());
```

Comment:
almost everything in this file right now is just adjusting whitespace, excluding this line and assertPatientInCurrentOrg() below.

Reply:
We already talked through the underlying issue, but FWIW, the lookup ID is frequently null. 😁